spec/factories/cards.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :card do
+    original_text 'Hello'
+    translated_text 'Привет'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.